gradle-module-plugin/src/main/kotlin/io/ia/sdk/gradle/modl/task/WriteModuleXml.kt

```diff
@@ -220,12 +215,32 @@ open class WriteModuleXml @Inject constructor(_objects: ObjectFactory) : Default
         return false
     }
 
-    private fun manifests(): List<ArtifactManifest> {
```

Comment:
Since it's private and used only from one place, no harm in changing the return type.